Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.cs

```diff
@@ -0,0 +1,118 @@
+            nameof(Header),
+            typeof(string),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(string.Empty));
+
+        /// <summary>
+        /// Gets or sets the title of this ribbon tab.
+        /// </summary>
+        public string Header
+        {
+            get => (string)GetValue(HeaderProperty);
+            set => SetValue(HeaderProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Footer"/> property.
+        /// </summary>
+        public static readonly DependencyProperty FooterProperty = DependencyProperty.Register(
+            nameof(Footer),
+            typeof(UIElement),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(new Border()));
+
+        /// <summary>
+        /// Gets or sets the <see cref="UIElement"/> to be displayed in the footer of the tab.
+        /// </summary>
+        public UIElement Footer
+        {
+            get => (UIElement)GetValue(FooterProperty);
+            set => SetValue(FooterProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="IsContextual"/> property.
+        /// </summary>
+        public static readonly DependencyProperty IsContextualProperty = DependencyProperty.Register(
+            nameof(IsContextual),
+            typeof(bool),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(false));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this tab is contextual.
+        /// </summary>
+        public bool IsContextual
+        {
+            get => (bool)GetValue(IsContextualProperty);
+            set => SetValue(IsContextualProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="OverflowButtonAlignment"/> property.
+        /// </summary>
+        public static readonly DependencyProperty OverflowButtonAlignmentProperty = DependencyProperty.Register(
+            nameof(OverflowButtonAlignment),
+            typeof(HorizontalAlignment),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(HorizontalAlignment.Left));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this tab is contextual.
+        /// </summary>
+        public HorizontalAlignment OverflowButtonAlignment
+        {
+            get => (HorizontalAlignment)GetValue(OverflowButtonAlignmentProperty);
+            set => SetValue(OverflowButtonAlignmentProperty, value);
+        }
+
+        /// <inheritdoc/>
+        protected override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+
+            _primaryItemsControl = GetTemplateChild("PrimaryItemsControl") as ItemsControl;
+            if (_primaryItemsControl != null)
+            {
+                _primaryItemsControl.HorizontalAlignment = HorizontalAlignment.Stretch;
```

Comment:
It'd probably be better to setup a Binding here so that the binding flows through, but I just learned how to do this and haven't added my hacky helper yet... 🤔 May need to make a follow-up note later for myself. 🙂

Reply:
I can't remember exactly why, but there was a reason why I hardcoded it. It's not a value that should be modified by the user.

Reply:
I remember why. `_primaryItemsControl.HorizontalAlignment` determines the alignment of the commands in the CommandBar. Now that I'm thinking about it, that would be a useful thing to expose (particularly for localization in the future). I've created the `CommandAlignment` property on `TabbedCommandBar`.